src/Components/Publishing/Header/BasicHeader.tsx

```diff
@@ -0,0 +1,148 @@
+import * as React from 'react'
+import { Col, Grid, Row } from 'react-styled-flexboxgrid'
+import styled, { css } from 'styled-components'
+import { pMedia as breakpoint } from "../../Helpers"
+import { Share } from "../Byline/Share"
+import { articleHref, getAuthorByline, getDate } from "../Constants"
+import { Fonts } from "../Fonts"
+
+interface Props {
+  article: any
+  isMobile?: any
+  leadParagraph?: any
+  layout?: any
+  title: any
+  vertical?: any
+}
+
+export const BasicHeader: React.SFC<Props> = (props) => {
+  const {
+    article: {
+      contributing_authors,
+      lead_paragraph,
+      published_at,
+      slug,
+    },
+    title,
+    vertical
+  } = props
+
+  return (
+    <Container>
+      <Grid fluid>
+        <Row>
+          <Col xs sm md lg>
+            <Vertical>
+              {vertical}
+            </Vertical>
+          </Col>
+        </Row>
+        <Row>
+          <Col xs sm md lg>
+            <Title>
+              {title}
+            </Title>
+          </Col>
+        </Row>
+        <Row around='xs' center='xs'>
+          <Col xs={12} sm={9} md={9} lg={9}>
+            <Row>
+              <Col xs sm md lg>
+                <LeadParagraph dangerouslySetInnerHTML={{
+                  __html: lead_paragraph
+                }} />
+              </Col>
+            </Row>
+            <Row>
+              <Col xs={4} sm={4} md={4} lg={4}>
+                <Author>
+                  By {getAuthorByline(contributing_authors)}
+                </Author>
+              </Col>
+              <Col xs={4} sm={4} md={4} lg={4}>
+                <Date>
+                  By {getDate(published_at, 'basic')}
+                </Date>
+              </Col>
+              <Col xs={4} sm={4} md={4} lg={4}>
```

Comment:
@kanaabe / @eessex - one thing that needs to happen somewhat urgently is the implementation of a grid system throughout all publishing layouts. I fear there might be some real difficulty later on if this doesn't happen -- but if it does, it will make things much less fragile overall. 

The above (pretty much) works, but there are subtle clashes between the responsive breakpoints and the grid standards which are noticeable when one drags a browser around, and we need to be sure to double-check cross-browser. It's the best that I can do for right now though.

Reply:
(As a side note, I really miss working with https://react-bootstrap.github.io/components.html#grid, and this component library in general. Have never come across anything thats easier to use. The grid system above is 🤔)